### PR TITLE
Add silent and update emp

### PR DESCRIPTION
**Pull request by iyZAIRI**

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -100,13 +105,40 @@ async def feedback(ctx, *, message):
...
+    """
     connection = sqlite3.connect('db/kolulu.db')
     with closing(connection) as db:
         statement = 'INSERT INTO feedback (user_id, feedback) VALUES (?, ?)'
         db.cursor().execute(statement, (ctx.author.id, message))
         db.commit()
 
     await ctx.send(f'Feedback received!')
+@bot.command()
+@commands.has_permissions(administrator=True)
```
Comment (nnguyen259) [inline bot.py:121]: Should be guild_only as well

File: bot.py [hunk 2]
```diff
@@ -68,14 +68,19 @@ async def on_command_error(ctx, error):
     msg = "".join(traceback.format_exception(type(error), error, error.__traceback__))
     logger.error(msg)
     logger.error(f'Error caused by user {ctx.author} running the command {ctx.message.content}')
-
     connection = sqlite3.connect('db/kolulu.db')
+    guildId = ctx.guild.id
```
Comment (nnguyen259) [inline bot.py]: Need a case when the error happens during dm

File: sql/002_v1.sql [hunk 3]
```diff
@@ -8,4 +8,10 @@ CREATE TABLE feedback2 (
 
 INSERT INTO feedback2(id, user_id, feedback) SELECT * FROM feedback;
 DROP TABLE feedback;
-ALTER TABLE feedback2 RENAME TO feedback;
\ No newline at end of file
+ALTER TABLE feedback2 RENAME TO feedback;
+
+CREATE TABLE "silence" (
+	"server_id"	INTEGER NOT NULL,
+	"silent"	INTEGER NOT NULL,
```
Comment (nnguyen259) [inline sql/002_v1.sql:15]: This could be Boolean instead of integer

**Review by nnguyen259 — COMMENTED** [review]
Another TODO: Remove the upgrades folder in the sql directory. Make sure the bot still works and still update its own db

File: bot.py [hunk 4]
```diff
@@ -68,14 +68,22 @@ async def on_command_error(ctx, error):
...
+            db.commit()
+            statement = 'SELECT silent FROM silence WHERE server_id = ?'
+            cursor = db.cursor()
+            cursor.execute(statement, (guildId,))
+            result = cursor.fetchone()
+            if result[0]==0:
+                await ctx.send(f'Error: {error}')
+            db.commit()
+    except:
+        await ctx.send(f'Error: {error}')
```
Comment (nnguyen259) [inline bot.py]: We still want to log the error into the db even in dm. Maybe add a boolean that will get switched during the except block? And only do the silent check if the boolean doesn't get switched

**Review by nnguyen259 — APPROVED** [review]

**Review by elee-github — APPROVED** [review]

**Pull request merged**